pages/common/g++.md

```diff
@@ -19,3 +19,11 @@
 - Include libraries located at a different path than the source file:
 
 `g++ {{path/to/source.cpp}} -o {{path/to/output_executable}} -I{{path/to/header}} -L{{path/to/library}} -l{{library_name}}`
+
+- Compile and link multiple source code files into executable binary:
+
+`g++ -c {{path/to/source_1.cpp}} {{path/to/source_2.cpp}} && g++ -o {{path/to/output_executable}} {{path/to/source_1.o}} {{path/to/source_2.o}}`
```

Comment:
Why would you compile and link the files separately? Wouldn't `g++ {{path/to/source1.cpp}} {{path/to/source2.cpp}} -o {{path/to/executable}}` also work?

Reply:
I wrote them seperately just in case someone needs the raw compiled file without linking.

Reply:
Honestly I am not opposed to this example as it hints the user about distinct compile/link stages :)